Reference/V9-Config/RuntimeMinificationSettings/index.md

```diff
@@ -15,45 +15,49 @@ This section allows you to configure the runtime minifications (defaults shown),
 "Umbraco": {
   "CMS": {
     "RuntimeMinification": {
-      "UseInMemoryCache": false,
-      "CacheBuster": "Version"
+      "UseInMemoryCache": true,
+      "CacheBuster": "Timestamp"
```

Comment:
This section is intended to show the defaults so it needs to be reverted. See https://github.com/umbraco/Umbraco-CMS/blob/79fdb9a876a822f8662679711f5f801f3159ff42/src/Umbraco.Core/Configuration/Models/RuntimeMinificationSettings.cs for ref